### DISCUSSION TITLE
OpenVPN for Android French translation

### DISCUSSION THREAD

**Discussion by bubuleur** [schwabe/ics-openvpn#1549] [Configuration problems]
Hello,
I come to you about the French translation of your application (OpenVpn for android) I downloaded the latest version and there are a lot of untranslated words in the applications, I looked on crowdin and the sentences are translated why are they not in the application?
Thank you in advance for your answer

**Comment by schwabe** [comment]
I haven't updated the translations in a while since dealing with the all spam and false trnaslations on crowdin is a real hassle.

**Reply by bubuleur** [reply]
> I haven't updated the translations in a while since dealing with the all spam and false trnaslations on crowdin is a real hassle.

Ok thank you for your answer I can help you with that
Is it possible to get a beta for me to test?
